wger/exercises/api/serializers.py

```diff
@@ -25,6 +25,24 @@
 )
 
 
+class ListAllSerializer(serializers.ModelSerializer):
+    '''
+    Special Exercise List serializer
+    '''
+    my_url = serializers.SerializerMethodField('get_absolute_url')
+
+    @classmethod
+    def get_absolute_url(cls, obj):
+        '''
+        Returns the canonical URL to view an exercise
+        '''
+        return "/api/v2/exercise/special/{}".format(obj.id)
```

Comment:
@Manorlds-Eaglespark I like how you set the exercise id with the absolute_url. This gives your implementation a cleaner finish as compared to before.  So a user can get all information for an exercise with its descriptive name and corresponding redirect